MakeMsix.targets

```diff
@@ -0,0 +1,78 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Define the property MakeMsixVerbosity=high for additional logging (default=normal) -->
+  <PropertyGroup>
+    <MakeMsixVerbosity Condition="'$(MakeMsixVerbosity)' == ''">normal</MakeMsixVerbosity>
+    <MakeMsixSigned Condition="'$(MakeMsixSigned)' == ''">true</MakeMsixSigned>
+  </PropertyGroup>
+
+  <Target Name="MakeMSIX"
+    Inputs="@(MakeMsixInputs);@(MakeMsixInputsWithLocations);@(MakeMsixInputAssets);$(MakeMsixOutputFilename)"
```

Comment:
How are MakeMsixInputs and MakeMsixInputsWithLocations different?  Is it the same list but one does not have a path component?

Reply:
Also, why is the output an input?

Reply:
> How are MakeMsixInputs and MakeMsixInputsWithLocations different? Is it the same list but one does not have a path component?

Path and/or filename

Most uses copy a file to the package root with the same name. A few exception cases need to copy a file to a subdir, or to a different filename, or both.

I tried adding destination metadata to _some_ input files but couldn't figure out syntax to not choke on that. So I caved and made 2 different properties. A bit ugly but not that bad all things considered, and time's run well over for this. If there's a more elegant solution it can be revised in a future PR.

Reply:
> Also, why is the output an input?

Because...when it comes to msbuild I'm the one-eyed man (in the kingdom of the blind)? ;-)

Should be

```
  <Target Name="MakeMSIX"
    Inputs="@(MakeMsixInputs);@(MakeMsixInputsWithLocations);@(MakeMsixInputAssets)"
    Outputs="$(MakeMsixOutputFilename)"
    AfterTargets="AfterBuild">
```

?